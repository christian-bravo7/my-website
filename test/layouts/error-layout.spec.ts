import ErrorLayout from '@/layouts/error-layout.vue';
import { mount } from '@vue/test-utils';

describe('error layout', () => {
  it('should render correclty', () => {
    const wrapper = mount(ErrorLayout);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
