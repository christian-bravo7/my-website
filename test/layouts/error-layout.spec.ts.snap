// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error layout should render correclty 1`] = `
<main class="min-h-screen flex items-center p-4 bg-gray-50 dark:bg-blue-600">
  <nuxt class="z-10"></nuxt>
  <errormovablefigures></errormovablefigures>
</main>
`;
